<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="screen1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="screen2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>461</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>331</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>Wszyscy klienci</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>Najbogatszy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>Najbiednijszy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>Dadaj do salda</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>Zmień dane klienta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button7">
       <property name="text">
        <string>wyjdź</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wczytaj">
       <property name="text">
        <string>Wczytaj</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>button7</zorder>
    <zorder>button1</zorder>
    <zorder>button2</zorder>
    <zorder>button4</zorder>
    <zorder>button6</zorder>
    <zorder>wczytaj</zorder>
    <zorder>button3</zorder>
   </widget>
   <widget class="QPlainTextEdit" name="screen3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="screen4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz numer konta lub liczbę klientów</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz zmienną</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
